environs/config/config.go

```diff
@@ -177,10 +178,16 @@ const (
 	// FanConfig defines the configuration for FAN network running in the model.
 	FanConfig = "fan-config"
 
-	// CloudInitUserDataKey is the key to specify cloud-init yaml the user wants to add
-	// into the cloud-config data produced by Juju when provisioning machines.
+	// CloudInitUserDataKey is the key to specify cloud-init yaml the user
+	// wants to add into the cloud-config data produced by Juju when
+	// provisioning machines.
 	CloudInitUserDataKey = "cloudinit-userdata"
 
+	// ContainerInheritProperiesKey is the key to specify a list of properties
+	// to be copied from a machine to a container during provisioning.  The
+	// list will be comma separated.
```

Comment:
Can we add a TODO to use a schema type of List (we are currently working on extending support for config attrs that support list and map types in the schema)

Reply:
done